### ISSUE TITLE
Remember the old install path when updating KVIrc on Windows

### ISSUE THREAD

**Issue by Heufneutje** [kvirc/KVIrc#1995]
When running KVIrc's setup on Windows, it recognizes that a previous version is already installed and uninstalls this before installing the new version. However, when installing the new version it defaults back to `%programfiles%\KVIrc`, rather than the path where KVIrc was installed before.

I opted not to fill the issue template since this is more of a feature request, but I'll add my executable information regardless.

```
KVIrc 4.9.2 'Aria'

Runtime Info:
System name: Windows Release : 6.2.9200
System version: Windows 8 (x64) (Build 9200)
Architecture: Intel64 Family 6 Model 42 Stepping 7
Qt version: 5.5.1
Qt theme: windowsvista

Build Info:
Build date: 2016-06-01 16:20:41 UTC
Sources date: 20160601
Revision number: git-6693-gfd8f4cd
System name: Windows-6.3.9600
CPU name: AMD64
Build command: C:/Program Files (x86)/CMake/bin/cmake.exe
Build flags: 
   MANDIR=share/man
   CMAKE_INSTALL_PREFIX=C:/projects/kvirc/build/release
   Threads=Win32
Compiler name: C:/Program Files (x86)/Microsoft Visual Studio 12.0/VC/bin/amd64/cl.exe
Compiler flags: /DWIN32 /D_WINDOWS /W3 /GR /EHsc
Qt version: 5.5.1
Features: IRC, IPv6, Crypt, SSL, IPC, Transparency, Webkit, Perl, Python, Enchant, Qt5, KVS
```

**Comment by un1versal** [comment]
yep thats been an annoyance each time I upgrade the windows build to test something since specifically dont want the default dir to be used

+1

**Comment by un1versal** [comment]
here seems simple removing https://github.com/kvirc/KVIrc/blob/a589f834e10f8767cbfb7ac26485cfcd07ff2214/cmake/KVIrc.nsi.cmake#L100 should do it

**Comment by un1versal** [comment]
once appveyor built windows executables out of https://github.com/kvirc/KVIrc/pull/1996 you can test it and **provide feedback**

**Comment by Heufneutje** [comment]
Doesn't seem to work for me. Still uses the default path when installing the new version.

**Comment by un1versal** [comment]
The problem here is currently this is hardcoded for new installs https://github.com/kvirc/KVIrc/blob/97cb75abf7e0f06c0f4820d73f8696e816e76b5e/cmake/KVIrc.nsi.cmake#L26

Im not sure if there's a way to disregard that this at re-installs especially since we force uninstall old version before installing the new one which looks as if ts counted as new install.

It would be nice thought, maybe someone whos more familiar with nsis can come up with some solution.

**Comment by un1versal** [comment]
Does anyone know if NSI supports this?
